public/locales/en-US/grafana.json

```diff
@@ -2534,6 +2534,9 @@
       }
     },
     "details": {
+      "connections-tab": {
+        "description": ""
```

Comment:
why are we adding an empty translation?

Reply:
Running `make i18n-extract` removes the value?

Reply:
> To update phrases in any translated language, edit the phrase in Crowdin

I think once the PR is merged and deployed I have to update the value in Crowdin.